stdlib/3/_thread.pyi

```diff
@@ -1,15 +1,22 @@
 # Stubs for _thread
 
-# NOTE: These are incomplete!
+from typing import Any, Callable, Dict, NoReturn, Tuple
 
-from typing import Any
+error = RuntimeError
 
 def _count() -> int: ...
 _dangling = ...  # type: Any
 
 class LockType:
-    def acquire(self) -> None: ...
+    def acquire(self, waitflag: int = ..., timeout: int = ...) -> None: ...
     def release(self) -> None: ...
+    def locked(self) -> bool: ...
 
+def start_new_thread(function: Callable[..., Any], args: Tuple[Any, ...], kwargs: Dict[str, Any] = ...) -> int: ...
+def interrupt_main() -> NoReturn: ...
```

Comment:
If I understand the docs correctly, this causes the main thread to interrupt, but can be called from subthreads. So I'd expect it to return (`None`?).

Reply:
Yes:
```
static PyObject *
thread_PyThread_interrupt_main(PyObject * self, PyObject *Py_UNUSED(ignored))
{
    PyErr_SetInterrupt();
    Py_RETURN_NONE;
}
```